training/distributed_training/pytorch/data_parallel/yolov5/yolov5.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Branding - consistently use one name formatting of <code>YOLOv5</code> versus <code>Yolo-v5</code>, etc. It looks like the official version is <code>YOLOv5</code>, see https://docs.ultralytics.com/ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>